mycli/main.py

```diff
@@ -1070,6 +1075,32 @@ def cli(database, user, host, port, socket, password, dbname,
             else:
                 click.secho(alias)
         sys.exit(0)
+    if list_ssh_config:
+        if not paramiko:
+            click.secho(
+                "Cannot use SSH transport because paramiko isn't installed, "
+                "please install paramiko or don't use --list-ssh-config=",
```

Comment:
This message sounds a bit harsh. 

Maybe say, "This features requires paramiko. Please install paramiko and try again."

Reply:
Sounds good.